health/notifications/gotify/README.md

```diff
@@ -0,0 +1,53 @@
+<!--
+title: "Send notifications to Gotify"
+description: "Send alerts to your Gotify instance when an alert gets triggered in Netdata."
+sidebar_label: "Gotify"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/health/notifications/gotify/README.md
+-->
+
+# Send notifications to Gotify
+
+[Gotify](https://gotify.net/) is a self-hosted push notification service created for sending and receiving messages in real time.
+
+First you have to create an application token in the Gotify WebUI. After you have generated a token, edit
+`health_alarm_notify.conf` on your system, by running the following from your [config
+directory](/docs/configure/nodes.md):
+ 
+```bash
+./edit-config health_alarm_notify.conf
+```
+
+Change the variable `GOTIFY_APP_TOKEN` with the application token you generated in Gotify. You have to also change
+`GOTIFY_APP_URL` to point to your Gotify instance.
+
+```conf
+SEND_GOTIFY="YES"
+
+# Application token
+# Gotify instance url
+GOTIFY_APP_TOKEN=Ar-6M777HvDPylv
```

Comment:
Hello @coffeegrind123 ,

I think it is better to keep this `GOTIFY_APP_TOKEN` without any value, why are you setting this value here?

Reply:
It is just documentation

Reply:
I understand this is our documentation, and I think that if we keep this value, it is possible some users will think this is a value that needs to be added to `health_alarm_notify.conf`, this is the motive I suggested to remove it.

If you want to specify a format that the variables need to have, I suggest to create another paragraph explaining it.

Reply:
Sorry, I will fix this

Reply:
@thiagoftsm the sentence is one line above the example

> Change the variable `GOTIFY_APP_TOKEN` with the application token you generated in Gotify. You have to also change `GOTIFY_APP_URL` to point to your Gotify instance.

Reply:
@ilyam8 we also have token for telegram, but we are using `111122223:....` that is something invalid. I never used `Gotify`, but when I saw the value, it appeared a valid value.